configmap-puller/cmd/configmap-puller/main.go

```diff
@@ -121,7 +121,7 @@ func watchConfigMap(ctx context.Context, clientset *kubernetes.Clientset, name,
 
 			case t := <-ticker.C:
 				log.Println("tick at:", t)
-				timeoutCtx, _ := context.WithTimeout(ctx, 10*time.Second)
+				timeoutCtx, _ := context.WithTimeout(ctx, 5*time.Second)
```

Comment:
Can the timeout stay the same? The ticker was shortened to reduce the delay between ConfigMap refreshes but does the timeout really need to be changed since this the max time to wait for k8s to respond?